modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/deployer/RemoteAppEntryDeployerImpl.java

```diff
@@ -120,9 +128,9 @@ private String _getPortletId(RemoteAppEntry remoteAppEntry) {
 			).put(
 				"com.liferay.portlet.instanceable",
 				remoteAppEntry.isInstanceable()
-			).put(
-				"javax.portlet.display-name",
-				remoteAppEntry.getName(LocaleUtil.US)
+			//).put(
```

Comment:
Do we want to leave this code commented?

Reply:
OMG :scream:

Reply:
:thinking: I don't have that in my final version (in local)...

Reply:
OK, Brian removed it in a following commit (included in this PR)

Reply:
I'll leave it there so that I can be ashamed forever :bowtie: